### PR TITLE
Fixes for Symfony 2.7 LTS

### PR DESCRIPTION
**Pull request by korotovsky**
* Removed all usages of `SecurityContext` and `SecurityContextInterface`
 * `SecurityContextInterfaces` is replaced to `TokenStorageInterface` in `AbstractOpenIdAuthenticationListener`
 * Cleaned imports
 * `pattern` is replaced to `path` in routing configs.

### CODE REVIEW

**Comment by korotovsky** [comment]
@makasim This PR looks good.

File: README.markdown [hunk 1]
```diff
@@ -22,6 +22,8 @@ The bulk of the documentation is stored in the `Resources/doc/index.md` file in
 
 [Read the Documentation for master](Resources/doc/index.md)
 
+[Read the Documentation for 1.4.0 (for Symfony 2.7.x)](https://github.com/formapro/FpOpenIdBundle/blob/1.4/Resources/doc/index.md)
```
Comment (makasim) [inline README.markdown]: 2.7.0

Comment (korotovsky) [inline README.markdown]: Do you mean `(for Symfony 2.7.0)` ?

Comment (makasim) [inline README.markdown]: Nope, the branch will be 2.7-dev and the tag of the bundle 2.7.0

Comment (korotovsky) [inline README.markdown]: Okay, got it.

**Pull request merged**